server/src/uuid/model/taxonomy_term/messages.rs

```diff
@@ -103,3 +109,35 @@ pub mod taxonomy_term_move_mutation {
         }
     }
 }
+
+pub mod taxonomy_term_create_mutation {
+    use super::*;
+    use crate::instance::Instance;
+    use crate::uuid::Uuid;
+    use crate::vocabulary::model::TaxonomyType;
+
+    #[derive(Deserialize, Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct Payload {
+        pub user_id: i32,
+        pub taxonomy_type: TaxonomyType,
+        pub parent_id: Option<i32>,
```

Comment:
```suggestion
        pub parent_id: i32,
```

let's not create another root :smile: (this should only be done by db migrations...)

Reply:
Alright, good. 
In legacy there is the [following](https://github.com/serlo/serlo.org/blob/main/packages/public/server/src/module/Taxonomy/src/Manager/TaxonomyManager.php#L137) though, but using a migration is even better.

How would we version the migration? Should we implement it in DB layer?

Reply:
> How would we version the migration? Should we implement it in DB layer?

Currently by https://github.com/serlo/serlo.org/tree/main/packages/migrations For shuting down legacy serlo.org we need to move this source code out. However this will be on of the last steps